SparkPost/2web2ui
# SA-1825: 5 and 15 minute precision are no longer the default precisions

### What Changed
 - Rollups precision select 1 min when range is up to 8 hours
 - Rollups precision select hour when range is up to 10 days (unchanged)
 - Rollups no longer recommend 5/15 minute precision (but are still options)

### How To Test
 - On account with rollups enable, select date range below 8 hours
 - Recommended precision should be 1min
 - Select precision above 8 hours, below 10 days
 - Precision should be hour
 - Verify that 5/15 minute precision still selectable
 - Verify that behavior is correct in OG theme 😢 

### To Do
- [ ] Does this need to apply to non-rollup metrics?
- [ ] Verify behavior of precision in metrics
